src/proto/grpc/lookup/v1/rls_config.proto

```diff
@@ -0,0 +1,225 @@
+// Copyright 2020 The gRPC Authors
```

Comment:
We don't need this proto.  We don't use it for anything.

Reply:
And then grpc_proto should get rid of this instead?

Reply:
No, it's needed in grpc/grpc-proto.  It's used from there in systems other than gRPC.  But gRPC itself doesn't use it right now, because we take in this config in JSON form.

That having been said, we will start using this in the not-too-distant future when we start configuring RLS via xDS, so I guess it's okay to add it here, even though it's not being used right now.

Reply:
If `rls_config` is valid and will be used in grpc, I like to keep this.